lib/authz/project_rules_update_workflow.go

```diff
@@ -193,7 +193,7 @@ func (m *ESCancelUpdateProjectTagsTask) Run(
 }
 
 type ESStartProjectTagUpdaterTask struct {
-	esClient            EsClient
+	domainService       ProjectTaggedDomainService
```

Comment:
🎉

Reply:
What are the potential domain services using this? I wonder if a comment would be helpful.

Reply:
Currently nodmanager, ingest-servce, and compliance-service. Right, I did not think about the auth domain services.